### PR TITLE
INKO-46 - Managing Client Household Expenses

### PR DESCRIPTION
**Pull request by jumba2010**
## Description
Describe the changes made and why they were made instead of how they were made.

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [ ] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

File: app/scripts/routes.js [hunk 1]
```diff
@@ -1010,6 +1010,12 @@
             .when('/editAddress/:addrType/:addrId/:clientId', {
                 templateUrl: 'views/administration/EditAddress.html'
             })
+             .when('/addExpenses/:clientId', {
+                            templateUrl: 'views/administration/AddExpenses.html'
+                        })
```
Comment (hero78) [inline app/scripts/routes.js:1022]: These looks to be part of the client directory not admin ?

File: app/views/administration/EditExpenses.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<div ng-controller="EditExpensesController">
+    <form name="FieldConfiguration" novalidate="" class="form-horizontal well" >
```
Comment (hero78) [inline app/views/administration/EditExpenses.html]: file should be moved to client's folder

File: app/views/administration/AddExpenses.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<div ng-controller="AddExpensesController">
```
Comment (hero78) [inline app/views/administration/AddExpenses.html]: file should be moved to client's folder

**Review by hero78 — CHANGES_REQUESTED** [review]

**Review by hero78 — DISMISSED** [review]
LGTM

**Review by Deepika1095 — DISMISSED** [review]

**Review by Deepika1095 — APPROVED** [review]

**Review by hero78 — APPROVED** [review]

**Pull request merged**